LondonDataServices.IDecide.Core.Tests.Unit/Services/Foundations/ConsumerStatuses/ConsumerStatusServiceTests.Exceptions.cs

```diff
@@ -0,0 +1,313 @@
+﻿// ---------------------------------------------------------
+// Copyright (c) North East London ICB. All rights reserved.
+// ---------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using EFxceptions.Models.Exceptions;
+using FluentAssertions;
+using LondonDataServices.IDecide.Core.Models.Foundations.ConsumerStatuses;
+using LondonDataServices.IDecide.Core.Models.Foundations.ConsumerStatuses.Exceptions;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+
+namespace LondonDataServices.IDecide.Core.Tests.Unit.Services.Foundations.ConsumerStatuses
+{
+    public partial class ConsumerStatusServiceTests
```

Comment:
Rename this file.  Should be `ConsumerStatusServiceTests.Add.Exceptions.cs`

Reply:
🦅 👁️